acceptance/lib/helper.rb

```diff
@@ -2,19 +2,28 @@ module Puppet
   module Acceptance
     module Reboot
       # due to a ruby bug the correct error code 1116, is returned modulo 256 = 92
-      SHUTDOWN_ABORT = 'cmd /c shutdown /a'
-      SHUTDOWN_NOT_IN_PROGRESS = 1116 % 256
+      WINDOWS_SHUTDOWN_ABORT = 'cmd /c shutdown /a'
+      WINDOWS_SHUTDOWN_NOT_IN_PROGRESS = 1116 % 256
```

Comment:
I have patched this in a commit I just pushed to #59.  If it is validated, I'll squash it into the main commit.